### DISCUSSION TITLE
App Clips

### DISCUSSION THREAD

**Discussion by dhoskins** [fastlane/fastlane#17044] [Q&A]
Does anyone have any hints for building a project using App Clips? My project is building and archiving correctly within Xcode, but when I use `gym` I get the following error:

```error: exportArchive: Provide a bundle identifier to select from available reformatters: <app.bundle.id> and <app.bundle.id.clip>```

**Comment by dhoskins** [comment]
I fixed this by passing the following parameter to `gym`:
```
    export_options: {
      distributionBundleIdentifier: <app.bundle.id>
    }
```

**Comment by eugenebiryukov** [comment]
If you want to use exportOptions.plist file, just add the following:
```
<key>distributionBundleIdentifier</key>
<string>your.bundle.id.to.export</string>
```
